starter-code/HeterogeneousRecyclerView/app/src/main/java/com/charlesdrews/heterogeneousrecyclerview/FootballRecyclerViewAdapter.java

```diff
@@ -15,8 +15,64 @@
  * Created by charlie on 11/14/16.
  */
 
+
 public class FootballRecyclerViewAdapter extends RecyclerView.Adapter {
 
+    List<BaseFootballObject> mObjectList;
+
+    public FootballRecyclerViewAdapter(List<BaseFootballObject> objectList) {
+        mObjectList = objectList;
+    }
+
+    @Override
+    public int getItemViewType(int position) {
+        if(mObjectList.get(position) instanceof Team){
+            return 2;
+        } else if(mObjectList.get(position) instanceof Player){
+            return 1;
```

Comment:
I recommend using int constants like PLAYER and TEAM rather than just plain int values, to make your code more readable.